src/app/components/bot/BotPreview.js

```diff
@@ -61,6 +63,13 @@ const BotPreview = ({ me, team }) => {
 
   const savedHistory = safelyParseJSON(window.storage.getValue('botPreviewMessageHistory'), []);
 
+  // FIXME: When the Feature Flag is removed, we probably should just use:
+  // const userRole = UserUtil.myRole(window.Check.store.getState().app.context.currentUser, TeamSlug)
```

Comment:
Why not just do it now?

Reply:
Because we have to be a super admin to see the bot preview page, but the `UserUtil.myRole` returns early if it's a super admin. So we never see the banner.